indexer/indexer.go

```diff
@@ -224,6 +225,7 @@ func ProcessDataFeed(csConsumer transports.Consumer, txFeed *txfeed.TxFeed, db *
 					if err != nil {
 						log.Error("Error creating a transaction", "err", err.Error())
 					}
+					log.Debug("megaStatement", "txt", megaStatement)
```

Comment:
We probably don't want to leave this in.

Reply:
Removed. Good catch.